test/net/sourceforge/kolmafia/objectpool/ConcoctionTest.java

```diff
@@ -37,6 +38,26 @@ public void steelOrgansAreSortedToTheTop() {
     assertThat(nonSteelOrgan.compareTo(steelMargarita), is(1));
   }
 
+  @Test
+  public void concoctionsCanBeComparedToThemselves() {
+    var steelMargarita = ConcoctionPool.get(ItemPool.STEEL_LIVER);
+    assertThat(steelMargarita, comparesEqualTo(steelMargarita));
+  }
+
+  @Test
+  public void copyComparesAsEqual() {
+    Concoction original = ConcoctionPool.get(ItemPool.STEEL_LIVER);
+    Concoction copy = new Concoction(original);
+    // Not same object
+    assertNotSame(original, copy, "Different objects not supposed to be ==");
+    // But are equals
+    assertEquals(original, copy, "Original and copy not equals.");
+    assertEquals(copy, original, "Copy and original.");
+    // Sort the same via compareTo
```

Comment:
The above were different tests when they were explicitly a.equals(b) and b.equals(a).  Maybe I should have ignored the suggestion to let the test framework do the compare?  Otherwise line 55 can and should be deleted.